### PR TITLE
Split some of the generated files

### PR DESCRIPTION
**Pull request by lhchavez**
This should make it easier to reason about the tests and stuff.

This is also a good intermediate stopping point before I start making deep surgery into the Dockerfile.

### CODE REVIEW

File: languages/forth.toml [hunk 1]
```diff
@@ -14,27 +14,28 @@ packages = [
...
 
 setup = [
-  "pushd /tmp",
-  "wget -O gforth-0.7.9.tar.xz http://www.complang.tuwien.ac.at/forth/gforth/Snapshots/0.7.9_20200716/gforth-0.7.9_20200716.tar.xz",
-  "tar -Jxf gforth-0.7.9.tar.xz",
-  "rm gforth-0.7.9.tar.xz",
-  "mv gforth-0.7.9_* gforth-0.7.9",
-  "cd gforth-0.7.9",
+  """
+  pushd /tmp
```
Comment (lhchavez) [inline languages/forth.toml:18]: funny story: some of the commands in some of the languages were being de-duplicated! so the `popd` at the very end of this file got commented-out, ha.

File: languages/nim.toml [hunk 2]
```diff
@@ -5,20 +5,21 @@ extensions = [
...
+  tar xf nim-$NIM_VER.tar.xz
+  rm -rf nim-$NIM_VER.tar.xz
+  chown -R root: nim-$NIM_VER/
+  cd nim-$NIM_VER
+  sh build.sh
+  ./bin/nim c koch.nim
+  ./koch tools
+  sh install.sh /usr/local/bin
+  cp ./bin/* /usr/local/bin
+  cd ..
```
Comment (lhchavez) [inline languages/nim.toml:20]: same with this `cd ..`. once we get to the world of the post-surgery Dockerfile, it'll be just a memory in our minds.

File: gen/index.js [hunk 3]
```diff
@@ -186,6 +198,34 @@ for (let target in objects) {
 	fs.chmodSync(path.join(dest, target), 0o755);
 }
 
+const perLangScripts = [
+	'self-test',
+];
+
+for (const perLangScript of perLangScripts) {
+	const dirname = path.join(dest, `share/polygott/${perLangScript}.d`);
```
Comment (dan-stowell) [inline gen/index.js:206]: Nice. Thank you.

File: languages/forth.toml (hunk 1 above)
Comment (dan-stowell) [reply languages/forth.toml:18]: Oh l-rd. That explains some oddly-breaking builds we had back over the summer.

**Review by dan-stowell — APPROVED** [review]
🍌

**Pull request merged**